### PR TITLE
Fix logout button

### PR DESCRIPTION
**Pull request by natmote**
I fixed the logout button (its link was broken), and changed the url scheme slightly.

### CODE REVIEW

File: fullhouse/dashboard/templates/dashboard.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
       <p>
 	<a href="#">User Settings</a>
 	<a href="#">House Settings</a>
-	<a href="welcome.html">Logout</a>
+	<a href="../">Logout</a>
```
Comment (j3ffml) [inline fullhouse/dashboard/templates/dashboard.html]: If this works, I'm surprised, it's a bit of a hack. the link should be to a logout view, which can redirect to the welcome/home page. Basically same thing you did with the 'home' view. That will make it easier to actually do the logout later.

File: fullhouse/dashboard/views.py [hunk 2]
```diff
@@ -2,6 +2,9 @@
 from django.http import HttpResponseRedirect
 from django.template import RequestContext
 
+def home(request):
+    return HttpResponseRedirect('/welcome.html')
+
```
Comment (j3ffml) [inline fullhouse/dashboard/views.py:7]: This is good. Should make one for `logout` as well.

**Comment by j3ffml** [comment]
I recommend making above changes and resubmitting pull request.

**Comment by natmote** [comment]
how's this?

File: fullhouse/dashboard/templates/welcome.html [hunk 3]
```diff
@@ -7,7 +7,7 @@
   <div id="logo">
     <img src="{{ STATIC_PREFIX }}main-logo.png" alt="logo" />
   </div>
-  <form id="login" action="login/" method="POST">
+  <form id="login" action="../login/" method="POST">
     {% csrf_token %}
```
Comment (j3ffml) [inline fullhouse/dashboard/templates/welcome.html:11]: this is good, but you should not need the '../'. the `action` value should match what will be read in `urls.py`  which is just plain 'login/'. Does it not work that way?

**Pull request closed without merging**